### PR TITLE
fix(api): do not include HEPA-UV in motor nodes

### PR DESCRIPTION
**Pull request by ahiuchingau**
# Overview
We should exclude hepa uv and its bootloader node from the motor nodes, so we don't get a timeout response fetching motor position from them.

### CODE REVIEW

File: api/src/opentrons/hardware_control/backends/ot3utils.py [hunk 1]
```diff
@@ -351,8 +351,13 @@ def motor_nodes(devices: Set[FirmwareTarget]) -> Set[NodeId]:
         NodeId.head_bootloader,
         NodeId.gripper_bootloader,
     }
+    hepa_nv_nodes = {
```
Comment (vegano1) [inline api/src/opentrons/hardware_control/backends/ot3utils.py]: typo?
hepa_uv_nodes

**Review by vegano1 — APPROVED** [review]
oh yeah, nice catch
Looks good, just left a small comment.

**Review by sfoster1 — APPROVED** [review]
Looks good, let's fix the typo tho

File: api/src/opentrons/hardware_control/backends/ot3utils.py (hunk 1 above)
Comment (ahiuchingau) [reply api/src/opentrons/hardware_control/backends/ot3utils.py]: lol yep u != n, who would have thought

**Pull request merged**